can_tools/scrapers/official/IL/__init__.py

```diff
@@ -0,0 +1,123 @@
+import pandas as pd
+import requests
+import us
+
+from ... import DatasetBaseNoDate
+from ..base import CMU, CountyData
+
+
+class IllinoisHistorical(DatasetBaseNoDate, CountyData):
```

Comment:
My other initial thought on first glance through - how would you feel about having separate files for each scraper (that aren't in __init__.py).

I think there's an opportunity to make the modules more specific to the source so it's clear what scrapers we have for a given state.

for example:
```
IL/
  __init__.py
  illinois_dph_scraper.py
```

Reply:
Also ok with that.

Most of the states only had one scraper, so I got lazy and instead of making a new file for a scraper and then bringing it into the `__init__.py` I just put code directly in `__init__.py`

Reply:
This is fine with me too.